time/datetime/include/datetime/datetime.h

```diff
@@ -68,7 +68,7 @@ int datetime_parse(const char *input, struct os_timeval *utctime,
  *
  * Returns 0 on success and OS_EINVAL on invalid input data
  */
-int clocktime_to_timeval(const struct clocktime *ct, struct os_timeval *tv);
+int clocktime_to_timeval(const struct clocktime *ct, const struct os_timezone *tz, struct os_timeval *tv);
```

Comment:
I'm not a big fan of changing APIs, but since this was only recently exported I think this is not a problem.

Reply:
better fixing it now before it's part of an official release